drools-wb-screens/drools-wb-scenario-simulation-editor/drools-wb-scenario-simulation-editor-kogito-client/src/main/java/org/drools/workbench/screens/scenariosimulation/kogito/client/popup/ScenarioSimulationCreationPopupView.java

```diff
@@ -117,12 +117,12 @@ public void onRuleClick(final ClickEvent event) {
     }
 
     protected void initializeDropdown() {
-        dmnAssetsDivElement.appendChild(scenarioKogitoCreationAssetsDropdown.getElement());
-        scenarioKogitoCreationAssetsDropdown.clear();
-        scenarioKogitoCreationAssetsDropdown.init();
-        scenarioKogitoCreationAssetsDropdown.initializeDropdown();
-        scenarioKogitoCreationAssetsDropdown.registerOnChangeHandler(() -> {
-            final Optional<KieAssetsDropdownItem> value = scenarioKogitoCreationAssetsDropdown.getValue();
+        dmnAssetsDivElement.appendChild(scenarioSimulationCreationAssetsDropdown.getElement());
+        scenarioSimulationCreationAssetsDropdown.clear();
+        scenarioSimulationCreationAssetsDropdown.init();
+        scenarioSimulationCreationAssetsDropdown.initializeDropdown();
+        scenarioSimulationCreationAssetsDropdown.registerOnChangeHandler(() -> {
+            final Optional<KieAssetsDropdownItem> value = scenarioSimulationCreationAssetsDropdown.getValue();
             selectedPath = value.map(KieAssetsDropdownItem::getValue).orElse(null);
```

Comment:
Could we check that DMN is valid here?

Reply:
@dupliaka I think this is not the correct point, here you have only the list of all DMNs files put inside the dropdown. I'll work for improve this part in https://issues.redhat.com/browse/KOGITO-1426